### PR TITLE
Update item in an existing wishlist

### PR DESCRIPTION
**Pull request by abhinavGupta16**
1. Update an existing item in a wishlist
2. Fix vagrant file does not need vagrant provision now
3. Fixed update wishlist and test case
4. Added test cases to update an existing item in a wishlist
5. updated readme.md

image

### CODE REVIEW

**Comment by yf1357** [comment]
We don't need to to update an item in the list. See the second screenshot I sent in the slack

File: service/service.py [hunk 1]
```diff
@@ -302,6 +302,42 @@ def update_wishlists(wishlist_id):
     app.logger.info("Wishlist with ID [%s] updated.", wishlist_id)
     return make_response(jsonify(message), status.HTTP_200_OK)
 
+######################################################################
+# UPDATE AN EXISTING ITEM IN A WISHLIST
```
Comment (Jesse1204) [inline service/service.py]: Professor said he wouldn't allow updating product names or product ids. We have nothing to update then. Please remove this method.

File: tests/test_service.py [hunk 2]
```diff
@@ -542,6 +543,136 @@ def test_update_wishlist_with_unsupported_media_type(self):
                          "415 Unsupported Media Type: Content-Type must be {}"
                          .format(app_type))
 
+    def test_update_item_in_wishlist_bad_request_2(self):
```
Comment (Jesse1204) [inline tests/test_service.py]: Same here for the test cases.

**Review by Jesse1204 — CHANGES_REQUESTED** [review]

File: service/service.py (hunk 1 above)
Comment (abhinavGupta16) [reply service/service.py]: I have removed the API for now

File: tests/test_service.py (hunk 2 above)
Comment (abhinavGupta16) [reply tests/test_service.py]: removed the test cases for now

**Review by yf1357 — CHANGES_REQUESTED** [review]
"PUT /wishlists/{wishlist_id}/items/{item_id} - Updates an existing item in the target wishlist" should be removed from readme

**Review by yf1357 — CHANGES_REQUESTED** [review]
image

Items are not displayed

**Comment by abhinavGupta16** [comment]
> image
> 
> Items are not displayed

resolved

**Review by yf1357 — APPROVED** [review]

**Review by xiafanzeng — APPROVED** [review]

**Review by Jesse1204 — APPROVED** [review]
LGTM

**Pull request merged**